addons/hr/models/res_config_settings.py

```diff
@@ -1,8 +1,6 @@
-# -*- coding: utf-8 -*-
 
 import threading
-from odoo import fields, models, api, _
-from odoo.exceptions import ValidationError
+from odoo import fields, models, api
```

Comment:
for alphabetical order :)
```suggestion
from odoo import api, fields, models
```